### ISSUE TITLE
Skip existing dependencies

### ISSUE THREAD

**Issue by sebastien-savalle** [cap-js-community/mtx-tool#38]
Hi,

The saas registry allows to skip existing dependencies when updating subscriptions. 
Would it be possible to add this option ? This would be very useful especially when we only want to add one dependency. 

<img width="1283" alt="image" src="https://github.com/cap-js-community/mtx-tool/assets/9685252/72fe550f-afb3-4d93-94f6-57d494358475">

Thanks and best regards,
Sebastien

**Comment by rlindner81** [comment]
Yes, that's a good idea. Should not be too difficult too code, I will have a look.

**Comment by rlindner81** [comment]
@sebastien-savalle I built you a pre-release for you. Can you check if this works as you expect?
[cap-js-community-mtx-tool-0.7.13.tgz](https://github.com/cap-js-community/mtx-tool/files/12258043/cap-js-community-mtx-tool-0.7.13.tgz)

**Comment by sebastien-savalle** [comment]
@rlindner81: Thanks a lot, it works as expected.
